src/class-wpml-beaver-builder-data-settings.php

```diff
@@ -56,4 +56,31 @@ public function get_fields_to_save() {
 	}
 
 	public function add_hooks(){}
+
+	/**
+	 * Adds slashes to data going into the database as WordPress
+	 * removes them when we save using update_metadata. This is done
+	 * to ensure slashes in user input aren't removed.
+	 *
+	 * Inspired by `\FLBuilderModel::slash_settings`
+	 *
+	 * @param mixed $data The data to slash.
+	 *
+	 * @return mixed The slashed data.
+	 */
+	private function slash( $data ) {
+		if ( is_array( $data ) ) {
+			foreach ( $data as $key => $val ) {
+				$data[ $key ] = $this->slash( $val );
+			}
+		} elseif ( is_object( $data ) ) {
+			foreach ( $data as $key => $val ) {
+				$data->$key = $this->slash( $val );
+			}
```

Comment:
@strategio always learning something new: I didn't know we can [loop through the visible properties of an object](https://www.php.net/manual/en/language.oop5.iterations.php).